pkg/expansion/types.go

```diff
@@ -0,0 +1,33 @@
+package expansion
+
+import (
+	mutationtypes "github.com/open-policy-agent/gatekeeper/v3/pkg/mutation/types"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// Expandable represents an expandable object and its metadata.
+type Expandable struct {
```

Comment:
Because object, oldObject, etc. only exist for webhook admission (not audit, gator, or any other enforcement point), we probably should not make these part of the interface. We can probably achieve the same result by modifying how the admission webhook invokes expansion.